pyramid/tests/test_integration.py

```diff
@@ -741,3 +743,29 @@ def _assertBody(body, filename):
     data = data.replace(b'\r', b'')
     data = data.replace(b'\n', b'')
     assert(body == data)
+
+
+class MemoryLeaksTest(unittest.TestCase):
+
+    def tearDown(self):
+        import pyramid.config
+        pyramid.config.global_registries.empty()
+
+    def get_gc_count(self):
+        last_collected = 0
+        while True:
```

Comment:
I don't know :) May be it possible in some cases with threads that works simultaneously with this test. Then GC will collects different count of objects in each step of cycle.